autoprotocol/instruction.py

```diff
@@ -57,9 +57,11 @@ def _as_AST(self):
         """
         # informatics is not serialized if empty.
         if self.informatics:
-            return dict(op=self.op, **self.data, informatics=self.informatics)
+            return_dict = dict(op=self.op, **self.data, informatics=self.informatics)
```

Comment:
nit: I think it was fine as is since dict() is called along with the return. You could add typehints to the class to see a summary output when referencing

Reply:
this was more of a stylistic change. I personally don't like returning in if statements